workflow-ui/compose/build.gradle.kts

```diff
@@ -13,6 +13,10 @@ android {
   composeOptions {
     kotlinCompilerExtensionVersion = libs.versions.androidx.compose.compiler.get()
   }
+  defaultConfig {
```

Comment:
Oops, didn't mean to merge this.

Reply:
No, not accident, required by a crummy old AVD version. But this ripples to break other things (how did I get a green PR?). Need to consider spreading the blast radius or else upgrading the AVDs.

@RBusarow @steve-the-edwards, what's your kneejerk?